src/util/Constants.js

```diff
@@ -16,7 +16,7 @@ exports.WSCodes = {
 
 const AllowedImageFormats = ['webp', 'png', 'jpg', 'jpeg', 'gif'];
 
-const AllowedImageSizes = Array.from({ length: 9 }, (e, i) => 2 ** (i + 4));
+const AllowedImageSizes = [...Array.from({ length: 9 }, (e, i) => 2 ** (i + 4)), 300, 600].sort((a, b) => a - b);
```

Comment:
Since the code is already complicated and hard to read, wouldn't it be worth to simply specify the numbers?

Reply:
This is internal code for private methods tho and it works perfectly. What I don’t think is needed is the sort function since we don’t send this array anywhere

Reply:
> 
> 
> Since the code is already complicated and hard to read, wouldn't it be worth to simply specify the numbers?

I would have rather just supplied the actual numbers as well — especially if Discord is going to continue adding sizes that don't fit the power of two rule.

On the other hand, as mentioned above this is only used internally.